config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -238,9 +238,14 @@ data:
       installFailingMessage: GCP Precondition Failed
     - name: GCPQuotaSSDTotalGBExceeded
       searchRegexStrings:
-      - "Quota \'SSD_TOTAL_GB\' exceeded"
+      - "Quota 'SSD_TOTAL_GB' exceeded"
```

Comment:
I'm unsure why the test for this succeeds. We are seeing clusters stuck provisioning while the error doesn't get caught - my best bet is on the extra backslashes here, but I can't figure out why the unit tests pass.

Reply:
I tried to track down the source of that error message and its printer does come with a salient caveat:

```
// Format formats the message as a string.
// This method is only intended for human consumption and ignores errors.
// Do not depend on the output being stable. Its output will change across
// different builds of your program, even when using the same version of the
// protobuf module.
```